src/MLUtils.jl

```diff
@@ -30,6 +33,9 @@ export batchsize,
 include("eachobs.jl")
 export eachobs
 
+include("parallel.jl")
+export eachobsparallel
```

Comment:
we should avoid exporting this until we settle for some interface (which could also be `eachobs(..., parallel=true)`)

Reply:
Will remove 👍